docs/help/LOOT-FAQs.html

```diff
@@ -149,3 +149,6 @@ <h3>Moving From BOSS</h3>
   <dt id="correct-load-order">How can the load order LOOT gives be right if BOSS gives a very different load order?</dt>
   <dd>There can be more than one correct load order for a given set of plugins. For example, some plugins may not conflict or depend on each other, so their relative orders wouldn't matter. The load order BOSS gives is hand-made to look 'neat' as well as work, so plugins from the same mod will usually get grouped together, but LOOT doesn't know which plugins belong to which mod, so it just puts them where it thinks they work.</dd>
 </dl>
+
+<!-- Add empty space at the bottom for scrolling convenience -->
+<p style="margin-left: 0;padding: 0 0 75em 0;border-width: 0px; border-color: white; border-style:none;"></p>
```

Comment:
I think this could be just `<div style="height: 75em"></div>`. However, having a fixed amount of space means that it'll only correctly position the last heading for certain browser viewport sizes, so most of the time it'll be wrong. I'm not sure this is an overall improvement.

Reply:
I've used this approach for the [regular expression doc](https://loot.github.io/docs/contributing/Regular-Expression-Regex.html), though in that case I've [used](https://github.com/loot/loot.github.io/blob/master/docs/contributing/Regular-Expression-Regex.html#L1210) 40em instead of 75em like in this PR.

We could also use your approach, or remove or replace the empty space if it doesn't turn out to work great. I just think this could help in spotting the right heading if we link to a specific list point of the FAQs, like if I would link to [How can LOOT not have unrecognised plugins?](https://loot.github.io/docs/help/LOOT-FAQs.html#unrecognised-plugins) as an example.